docs/schema/template-element-workflowconfig.md

```diff
@@ -105,20 +106,11 @@ The following sections describe attributes, child elements, and parent elements.
 
 ### Parent elements
 
-<table>
-<colgroup>
-<col width="100%" />
-</colgroup>
-<tbody>
-<tr class="odd">
-<td align="left"><p><span sdata="link"><a href="workflowconfig-element.md">WorkflowConfig Element</a></span></p></td>
-</tr>
-</tbody>
-</table>
+[WorkflowConfig Element](workflowconfig-element.md)
```

Comment:
Let's turn this HTML table into a single column Markdown table, something like this:
|Element|
|---|
|`WWorkflowConfig Element](workflowconfig-element.md)`|